w3af/core/data/context/context/html.py

```diff
@@ -133,12 +156,18 @@ class HTMLAttrQuoteGeneric(BaseContext):
     JS_PATTERN = re.compile('^ *javascript:', re.IGNORECASE)
     VB_PATTERN = re.compile('^ *vbscript:', re.IGNORECASE)
 
-    def __init__(self, payload, attr_name, attr_value):
+    # What about &NewLine;, &Tab;, etc?
+    JS_PROTOCOLS = {'javascript:', 'javascript&colon;',
```

Comment:
> Why not?

Browser expects `<a href="javascript:foo(2)">` not `<a href="javascript&colon;foo(2)">`